R/fit.R

```diff
@@ -101,15 +101,19 @@ spim_particle_filter <- function(data, pars, control,
 ##' @return A `mcstate_pmcmc` object
 ##'
 ##' @export
-spim_fit_run <- function(pars, filter, control) {
+spim_fit_run <- function(pars, filter, control, initial = NULL) {
   message("Running chains - this will take a while!")
   multiregion <- filter$nested
-  if (multiregion) {
-    initial <- replicate(control$n_chains,
-                         pars$mcmc$propose(pars$mcmc$initial(), "both", 1))
+  if (is.null(initial)) {
+    if (multiregion) {
+      initial <- replicate(control$n_chains,
```

Comment:
Decide in the with-tuning case if we want to do this somewhere else? interacts with how we might propagate the initial points (e.g., continuing chains or taking the best point over all chains)